### PR TITLE
Server switch

### PR DESCRIPTION
**Pull request by Sirzhatina**

### CODE REVIEW

File: Client/Client.Core/Client.Core.Static/ServerInfo.hpp [hunk 1]
```diff
@@ -7,9 +7,20 @@ namespace ServerInfo
 {
 /// Server address
 constexpr std::string_view address = "104.40.239.183";
```
Comment (fisyamaloy) [inline Client/Client.Core/Client.Core.Static/ServerInfo.hpp:9]: is it needed address here?

File: Client/Client.Core/Client.Core.Static/ServerInfo.hpp [hunk 2]
```diff
@@ -7,9 +7,20 @@ namespace ServerInfo
...
 /// you can change the needed server using two variables above
 /// also, test means "fresh" e.g. you are on master now
 /// production means based on release commit that is tagged.
-constexpr std::uint16_t port = production_server;
+namespace Port
+{
+    constexpr std::uint16_t test       = 65001;
+    constexpr std::uint16_t production = 65003;
+}
+constexpr std::uint16_t port = test_server;
```
Comment (fisyamaloy) [inline Client/Client.Core/Client.Core.Static/ServerInfo.hpp:25]: Here too?

File: Client/Client.Qt/Settings.cpp [hunk 3]
```diff
@@ -1,31 +1,27 @@
...
-Settings::Settings()
+Settings::Settings(const QString& filename, QSettings::Format format)
+    : QSettings(filename, format)
 {
-    settings = std::make_unique<QSettings>();
 }
 
-std::unique_ptr<Settings> Settings::instance = nullptr;
-
 Settings& Settings::getInstance()
```
Comment (fisyamaloy) [inline Client/Client.Qt/Settings.cpp:9]: I think, Inline would be great here

File: Client/Client.Core/Client.Core.Static/Client.cpp [hunk 4]
```diff
@@ -15,12 +15,12 @@ Client::~Client() { disconnectFromServer(); }
 
 bool Client::connectToServer(const std::string_view& host, const uint16_t port)
 {
-    if (host != ServerInfo::address)
+    if (host != ServerInfo::Address::remote && host != ServerInfo::Address::local)
```
Comment (Orange-hanter) [inline Client/Client.Core/Client.Core.Static/Client.cpp:18]: Not sure that this is a better way to validate input data. But it's OK for now.

File: Client/Client.Qt/ConnectionManager.hpp [hunk 5]
```diff
@@ -79,12 +79,14 @@ class ReceiverManager : public QObject
 class ConnectionManager : public Network::Client
 {
 public:
+    ConnectionManager() { configureConnectionProperties(); }
```
Comment (Orange-hanter) [inline Client/Client.Qt/ConnectionManager.hpp:82]: Better to keep one style. If all implementation is moved to cpp than all implementations should be moved to cpp file.

**Review by Orange-hanter — APPROVED** [review]

**Review by Orange-hanter — APPROVED** [review]

**Review by f1nal3 — APPROVED** [review]

**Pull request merged**